letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
+    "A"
+  elsif integer < 90 && integer >= 80
```

Comment:
if you go this route with a guard clause of sorts keep in mind that case statements and ranges are also an option.

```
case integer
when 80..90
  ...
end
```